js/spinner.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+(function () {
+    let conteiner = document.createElement("div");
+    conteiner.classList.add("container-spinner");
+    let spinner = document.createElement("div");
+    spinner.classList.add("spinner");
+    conteiner.appendChild(spinner);
+    let position = document.getElementsByTagName("body")[0];
```

Comment:
recomend write like this: let position = document.body;

Reply:
ready